docs/features/software-catalog/user-guide.md

```diff
@@ -0,0 +1,99 @@
+
+### Descriptor Components used to build the Catalog Graph
+
+**Entities:** An entity refers to a node in the graph that represents a distinct object, concept, or thing. Nodes are the fundamental building blocks of a graph database and are used to represent entities and their properties.
+
+![](../../assets/software-catalog/entity.svg)
+
+**Kinds:** These are broad categories used to group related entities. Kinds are used to provide a high-level categorization of entities, such as "service", "database", or "team". Kinds are often used to provide a way to filter entities in the catalog and to provide a high-level overview of the types of entities that are being managed.
+
+**Relations:** These are links between different entities in the catalog. Relations express the relationships between different entities, such as dependencies or ownership. Adopters can use relations to help users navigate the catalog and understand the relationships between different entities.
+
+**Spec:** A specification or "spec" is a schema that outlines the data structure of an entity in the Backstage catalog. It defines the properties, relationships, data types, and constraints for an entity, ensuring consistency and accuracy of data while allowing for easy sharing and consumption of data across components and plugins. Specs are useful when creating or extending entities and can help make data more reusable and interoperable. The spec section is fully customizable, and users can create their own components and plugins to render the information.
+
+**Types:** These are more specific categories used to classify entities within a given Kind. Types provide a more granular categorization of entities, such as "frontend-service" or "backend-service.". Types are often used to provide additional context and information about an entity and to help users understand the role and function of the entity within the broader system.
+
+**Annotations:** These key-value pairs can be attached to an entity in the catalog. They are typically used to add additional information or metadata to an entity. Annotations are often used to provide information that is used by automated tools or scripts and to provide further context to humans working with the entity or refer plugins to the external world.
+
+**Statuses:** These indicators represent an entity's current state. They are typically used to show whether an entity is healthy or not, whether it is up-to-date or out-of-date, or whether it is passing or failing various checks or tests.
```

Comment:
Is this actually true? I'm not sure that the up-to-date or out-of-date information is included in this. Happy to be wrong. This might be another chance to link to to the docs: https://backstage.io/docs/features/software-catalog/descriptor-format#common-to-all-kinds-status

Reply:
Agreed, this field is pretty "unstable", as in, it only has one single purpose now which is to show that the processing loop has some issue with the entity. We thought there would be more use cases for the field but that hasn't happened yet. Indeed it's probably best to not be too specific about this one, and instead link to the technical doc (or just leave it out of the list here perhaps?).

Reply:
Perfect so I'll remove this section